telegram/src/test/kotlin/com/github/kotlintelegrambot/types/TelegramBotResultTest.kt

```diff
@@ -61,7 +61,7 @@ class TelegramBotResultTest {
     }
```

Comment:
I would keep the previous test names. One test name or other is mostly a matter of taste, and is a change unrelated to the PR, so I would advocate to keep it as was. Otherwise, we could end up with back and forth changes in tests names that lead nowhere.

Reply:
What do you think than of separate pr with naming convention?
I belive that tests naming should follow one 
e.g. `Should do something when given` or `When given should do something`
This way we can detect some issues with architecrure or split tests with similar setups into nested classes
e.g.
```kotlin
@Nested
inner class `Should send MediaGroup with media group composed by a video(fileUrl), a photo(fileId)` {
    // shared fixture with response and media group
    @Test
    fun `when all mandatory arhuments is passed`() {
        // test
    }
    @Test
    fun `when all arguments is passed`() {
        // test
    }
```

Reply:
I don't have a strong opinion about tests naming. I used to follow the "should do something when given" naming but it sometimes ends up being too verbose and makes test names and tests harder to understand. Now I try to keep it simple in the name and just follow a `given-when-then`|`arrange-act-assert` structure in the test itself. I guess it's also a matter of taste and something to be agreed with all the people working in a project to be used in a consistent way (in this case the project is quite small and the people working on it is not too many, so I guess it's not that critical).

I like the nested tests though, we're not taking advantage of them and we should, good call 👍🏻

Reply:
Maybe than we should create issue with poll about it? Also it would help with writing contributing guide for project, it could decrease number of this **taste** cases
Thanks for review!)